src/platform/bouffalolab/BL616/NetworkCommissioningDriver.cpp

```diff
@@ -206,6 +206,9 @@ void BLWiFiDriver::ScanNetworks(ByteSpan ssid, WiFiDriver::ScanCallback * callba
     if (callback != nullptr)
     {
         mpScanCallback = nullptr;
+
+        memcpy(mScanSSID, ssid.data(), ssid.size());
```

Comment:
how do we know mScanSSID has sufficient space for ssid.size() amount of data?

Reply:
fix to verify SSID length prior to copying